### DISCUSSION TITLE
Installation (V3) failed on Ubuntu 20.04.4 LTS

### DISCUSSION THREAD

**Discussion by Chiller2019** [Kozea/Radicale#1415] [External Packaging]
Hello,

I would install Radicale on Ubuntu 20.04.4 LTS. (Clean/Fresh Machine)
Installed Python3 (Version: 3.8.10) and pip.

Freeze on (show -v): "import 'encodings.idna' # <_frozen_importlib_external.SourceFileLoader object at 0x7f5e79b1eac0>"

I dont know why. Anyone can test it and maybe have a answer?

**Comment by jonassmedegaard** [comment]
Ubuntu is based on Debian, where I maintain a packaging of Radicale, so I would guess that you can do without pip - instead simply run `sudo apt install radicale`.

**Comment by Chiller2019** [comment]
> Ubuntu is based on Debian, where I maintain a packaging of Radicale, so I would guess that you can do without pip - instead simply run `sudo apt install radicale`.

I know but this version is (think) v2 and very old, not up to date.

**Comment by jonassmedegaard** [comment]
Sorry, then I cannot help you.

v3 is included with Debian stable released that was released some time ago, and v3.1 is in Debian unstable, testing, and stable-backports: https://tracker.debian.org/radicale.

**Comment by lisbonjoker** [comment]
Ubuntu installed radicale 3.1.7 for me but however I cannot create systemd tried everything, got it running but it keeps on giving "A server error occurred.  Please contact the administrator."
```
Apr 21 22:48:03 env[13373]: [2022-04-21 22:48:03 +0100] [13373/Thread-2] [ERROR] An exception occurred during GET request on '/': [Errno 13] Permission denied
Apr 21 22:48:03 env[13373]: [2022-04-21 22:48:03 +0100] [13373/Thread-2] [ERROR] An exception occurred during GET request on '/favicon.ico': [Errno 13] Permission denied
Apr 21 22:54:22 env[15083]: [2022-04-21 22:54:22 +0100] [15083/Thread-3] [ERROR] An exception occurred during GET request on '/.web/': [Errno 13] Permission denied

```
Also it imported the old config from v2 rather than the new one, had to replace it.

**Comment by lisbonjoker** [comment]
Got a different error now
```
 An exception occurred during PROPFIND request on '/': [Errno 13] Permission denied
 ```

**Comment by pbiering** [comment]
Looks like directory/file permission problems, already solved? If not, become the user of the radicale process and check whether you can access directories and files.
